src/app/screens/DetailArticle/index.tsx

```diff
@@ -1,18 +1,30 @@
-import React, { useState } from 'react';
+import React, { useState, useCallback } from 'react';
 import { View, Image, TouchableOpacity, Text } from 'react-native';
+import { useSelector } from 'react-redux';
 import CustomText from '@components/CustomText';
-import { Navigation } from '@interfaces/navigation';
+import useNavigation from '@components/AppNavigator/helper';
+import { State } from '@interfaces/reduxInterfaces';
+import { Article } from '@interfaces/articlesInterface';
 import { formatDate } from '@utils/dateUtils';
 import icAddInactive from '@assets/TabBar/icAddpostInactive.png';
 import icAddActive from '@assets/TabBar/icAddpostActive.png';
 import icFavouriteInactive from '@assets/TabBar/icFavoriteInactive.png';
 import icFavouriteActive from '@assets/TabBar/icFavoriteActive.png';
 import icDefaultArticleImage from '@assets/icons/icDefaultArticleImage.jpg';
+import icDelete from '@assets/icons/icDelete.png';
+import icEdit from '@assets/icons/icEdit.png';
+import Routes from '@constants/routes';
 
 import styles from './styles';
 import testIds from './testIds';
 
-interface Props extends Navigation {}
+export interface Props {
```

Comment:
I think this interface is going to be repeated a lot, could you make a general one?

route: {
    params: {
      
    };
  };